mmselfsup/models/heads/mae_finetune_head.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from mmcls.models import LabelSmoothLoss
+from mmcv.cnn.utils.weight_init import trunc_normal_
+from mmcv.runner import BaseModule
+from torch import nn
+
+from ..builder import HEADS
+
+
+@HEADS.register_module()
+class MAEFinetuneHead(BaseModule):
```

Comment:
Head is currently redundant. Can `mae_pretrain_head.py`, `mae_finetune_head.py`, `mae_linearprob_head.py` be merged into one `mae_head.py` file?

Reply:
Thank you for your valuable comments. Since `mae_pretrain_head.py`, `mae_finetune_head.py` and `mae_linprobe_head` share different `forward` function, it's not convenient to merge them into one fun. If we merge them, we have to introduce extra params to control the `workflow` and the `forward` func will become too complex.

Reply:
If this is the case, I suggest writing these three classes into one `mae_head.py` file instead of three separate files.

Reply:
Or I think maybe it is better to merge finetune and linprobe heads and leave the pretrain head